pkgs/development/python-modules/fbtftp/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi }:
+buildPythonPackage rec {
+  pname = "fbtftp";
+  version = "0.4";
+
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qbfwk2qbbv8z0mf3hyykkp32lisv7g64w26ki7rf8nifj2lgnn6";
+  };
+
```

more standard layout
```suggestion
{ lib, buildPythonPackage, fetchPypi }:

buildPythonPackage rec {
  pname = "fbtftp";
  version = "0.4";

  src = fetchPypi {
    inherit pname version;
    sha256 = "0qbfwk2qbbv8z0mf3hyykkp32lisv7g64w26ki7rf8nifj2lgnn6";
  };

  doCheck = false;
```